src/features/Layout/Navigation/components/NavTray.tsx

```diff
@@ -24,15 +25,26 @@ const SIcon = styled(Icon)`
 
 export const NavTray = ({ tray, content }: { tray: INavTray; content: ReactNode }) => {
   const [isOpen, setOpen] = useState<boolean>(false);
+  const ref = useRef<HTMLDivElement>(null);
+
+  const { isMobile } = useScreenSize();
 
   const [isReady, clear, set] = useTimeoutFn(() => setOpen(false), 300);
 
+  const handleToggle = () => {
+    if (!isMobile) return;
+    setOpen(!isOpen);
+  };
+
+  useClickAway(ref, () => isMobile && setOpen(false));
+
   const handleOpen = () => {
+    if (isMobile) return;
     isReady() === false && clear();
```

Comment:
On mobile this already worked without the need of a `useClickAway`

Reply:
This way it closes when you hit a tray link 👍